lib/server.js

```diff
@@ -156,7 +157,6 @@ server.start = config => {
 
 server.cleanup = app => {
   if (app.stopped) {
-    console.log('twice')
```

Comment:
we can remove this `app.stopped` check entirely if we remove the `process.on('exit')` listener